.stylelintrc

```diff
@@ -80,5 +80,15 @@
         ]
       }
     ]
-  }
+  },
+  overrides: [
+    {
+      files: ['**/*.scss'],
+      customSyntax: 'postcss-scss'
+    },
+    {
+      files: ['**/*.vue'],
+      customSyntax: 'postcss-html'
+    }
+  ]
```

Comment:
see
- https://stylelint.io/migration-guide/to-14/
- https://github.com/postcss/postcss#syntaxes